addon/modifiers/toggle-modal.js

```diff
@@ -0,0 +1,14 @@
+import { modifier } from "ember-modifier";
+
+export default modifier(function toggleModal(
+  element,
+  [visible = false] = [] /*, hash*/
+) {
+  const isVisible = !!visible;
```

Comment:
Why the `!!`?

Reply:
Because the input is either null or an object, instead of converting it to a boolean in the template, I convert it here. Though not really necessary.

Reply:
I think you can also just use the `Boolean()` constructor.

Reply:
The linter throws a redundant Boolean call error, so I've left it out with the newest version.